Alexey/Text/count_vowels/main/count_vowels.py

```diff
@@ -0,0 +1,15 @@
+def main():
```

Comment:
Could you please create one more function that will return number of each vowel in the string?